clients/bigquery/tableid.go

```diff
@@ -31,6 +31,10 @@ func (ti TableIdentifier) Dataset() string {
 	return ti.dataset
 }
 
+func (ti TableIdentifier) EscapedTable() string {
```

Comment:
We should probably call this function in `FullyQualifiedName` too.

Reply:
sg, done